core/tests/test_utils.py

```diff
@@ -677,6 +678,57 @@ def save_new_valid_exploration(
         exp_services.save_new_exploration(owner_id, exploration)
         return exploration
 
+    def save_new_valid_exploration_with_states(
```

Comment:
Probably save_new_linear_exp_with_state_names_and_interactions() since you're not fully specifying the states...

(Bit of a mouthful, I know, but seems more accurate. Might be worth the trade-off?)

Reply:
Done.